File: apps/demo-android/app/src/main/java/com/cinacoin/demo/CinacoinDemoApp.kt
package com.cinacoin.demo

import android.app.Application
import com.cinacoin.appkit.config.AppKitConfig
import com.cinacoin.appkit.config.CinacoinAppKit
import com.cinacoin.appkit.auth.AuthManager

class CinacoinDemoApp : Application() {
    override fun onCreate() {
        super.onCreate()

        // Configure AppKit
        CinacoinAppKit.configure(
            AppKitConfig(
                projectId = "YOUR_PROJECT_ID",
                metadata = AppKitConfig.AppMetadata(
                    name = "Cinacoin Demo",
                    description = "Cinacoin Android SDK Demo",
                    url = "https://demo.cinacoin.com",
                    icons = listOf("https://cinacoin.com/icon.png")
                ),
                themeMode = AppKitConfig.ThemeMode.DARK
            )
        )

        // Configure Auth
        AuthManager.configure(projectId = "YOUR_PROJECT_ID")
    }
}

File: apps/demo-ios/CinacoinDemo/CinacoinDemoApp.swift
import SwiftUI
import CinacoinAppKitConfig
import CinacoinAppKitAuth

@main
struct CinacoinDemoApp: App {
    
    init() {
        // Configure AppKit
        CinacoinAppKit.configure(with: AppKitConfig(
            projectId: "YOUR_PROJECT_ID",
            metadata: AppKitConfig.AppMetadata(
                name: "Cinacoin Demo",
                description: "Cinacoin iOS SDK Demo",
                url: "https://demo.cinacoin.com",
                icons: ["https://cinacoin.com/icon.png"]
            ),
            themeMode: .dark
        ))
        
        // Configure Auth
        AuthManager.shared.configure(projectId: "YOUR_PROJECT_ID")
    }
    
    var body: some Scene {
        WindowGroup {
            ContentView()
        }
    }
}
